python/lsst_versions/_versions.py

```diff
@@ -384,28 +384,50 @@ def _process_version_writing(
     write_to: Optional[str] = None
     written = None
     if write_version:
-        write_to = _find_version_path(dir)
+        write_to = _find_version_path(dirname)
         if write_to is None:
             return "<unknown>", written
 
     # Find the version of HEAD and current directory.
     version: Optional[str] = None
     try:
-        version = find_lsst_version(dir, "HEAD")
+        version = find_lsst_version(dirname, "HEAD")
     except Exception:
         if not fallback:
             raise
     if version is None:
-        version = _find_version_from_metadata(dir)
+        version = _find_version_from_metadata(dirname)
         if version is None:
-            raise RuntimeError(f"Unable to find a version from Git or metadata within directory {dir}")
+            raise RuntimeError(f"Unable to find a version from Git or metadata within directory {dirname}")
 
     if write_version and write_to:
         _write_version(version, write_to)
 
     return version, write_to
 
 
+def get_lsst_version() -> str:
+    """Determine the version and return as string
+
+    Returns
+    -------
+    version : `str`
+        The version string.
+    """
```

Comment:
Please add a notes section saying that this will first look for a git version and then fall back to looking at package metadata.